locale/en_US/common.po

```diff
@@ -585,6 +585,9 @@ msgstr "Override"
 msgid "common.pagination"
 msgstr "{$start}-{$end} of {$total}"
 
+msgid "common.listSeparator"
```

Comment:
I found these missing locales in the error log

Reply:
I suspect it was replaced with `common.commaListSeparator` and `common.semicolonListSeparator`, at least I used those.

Reply:
Thanks! So I'll remove this locale and update the places which are still using the old one.